### ISSUE TITLE
Trilinos develop updates frozen?

### ISSUE THREAD

**Issue by spdomin** [trilinos/Trilinos#9797]
## Question
@trilinos

Greetings,

I see develop is stuck at the following SHA1:

1cb5bc6b9366977ecb143b7bcdcc5e1e94274f54 

Just making sure that develop is still the correct branch for the most up-to-date developer changes against which I should test nightly. 

best,

**Comment by jhux2** [comment]
This is a question for the @trilinos/framework team.  PR testing was not working last week ....

**Comment by jwillenbring** [comment]
Yes, we are working to restore PR testing. This is the correct branch for your testing.
